Lua/Lua.sublime-syntax

```diff
@@ -305,18 +305,19 @@ contexts:
     - include: else-pop
 
   variable-modifier:
-    - match: \<(?=\b(?:const|close)\b)
+    - match: '<'
       scope: punctuation.definition.modifier.begin.lua
       push:
         - variable-modifier-body
```

Comment:
I'd suggest to never use multi-push or multi-set to push/set only one context.

Reply:
Right, sorry. I forgot that one.

Reply:
That's why 4 eyes. :-)